packages/babel-preset/esm.js

```diff
@@ -0,0 +1,49 @@
+const envPreset = require('@babel/preset-env');
+const modulesPreset = require('@babel/preset-modules');
+const reactPreset = require('@babel/preset-react');
+const plugins = require('./plugins');
+
+function preset(api, options = {}) {
```

Comment:
maybe add a comment noting that this preset is subject to breaking changes (hence why it's not merged into the root preset)

Reply:
that would make me happy w/this being split out